addOns/commonlib/src/main/resources/org/zaproxy/addon/commonlib/internal/vulns/vulnerabilities.xml

```diff
@@ -342,7 +342,16 @@ Ensure that all failures in resource allocation place the system into a safe pos
 
 The most common type of a brute force attack in web applications is an attack against log-in credentials. Since users need to remember passwords, they often select easy to memorize words or phrases as passwords, making a brute force attack using a dictionary useful. Such an attack attempting to log-in to a system using a large list of words and phrases as potential passwords is often called a "word list attack" or a "dictionary attack". Attempted passwords may also include variations of words common to passwords such as those generated by replacing "o" with "0" and "i" with "1" as well as personal information including family member names, birth dates and phone numbers.
 	</desc>
-	<solution></solution>
+	<solution>Brute force attacks are among the hardest to prevent, but with thorough analysis, design, and multiple countermeasures, you could limit your system's exposure to them. The general strategy is to decrease the automation potential and credential discovery likelihood. Techniques to consider include:
```

Comment:
Are you the author of these solutions or are they based on some source?

Reply:
I’m the author of the presented information. Do you have any specific concerns? What they are?

Reply:
No concerns, but want to know who wrote it, if it was based on some source we would have to check it for changes.